_drafts/2017-08-24-FedData-release.md

```diff
@@ -0,0 +1,218 @@
+---
+name: FedData-release
+layout: post
+title: FedData - Getting assorted geospatial data into R
+date: 2017-08-24
+authors:
+  - name: Kyle Bocinsky
+    url: https://github.com/bocinsky
+categories:
+  - technotes
+tags:
+  - R
+  - package
+  - data-access
+  - packages
+  - spatial
+  - geospatial
+  - review
+  - onboarding
+  - FedData
+---
+
+The package [FedData](https://github.com/ropensci/FedData) is now part of [rOpenSci](https://ropensci.org/). FedData includes functions to automate downloading geospatial data available from several federated data sources (mainly sources maintained by the US Federal government).
+
+Currently, the package enables extraction from six datasets:
+
+-   The [National Elevation Dataset (NED)](http://ned.usgs.gov) digital elevation models (1 and 1/3 arc-second; USGS)
+-   The [National Hydrography Dataset (NHD)](http://nhd.usgs.gov) (USGS)
+-   The [Soil Survey Geographic (SSURGO) database](http://websoilsurvey.sc.egov.usda.gov/) from the National Cooperative Soil Survey (NCSS), which is led by the Natural Resources Conservation Service (NRCS) under the USDA,
+-   The [Global Historical Climatology Network (GHCN)](http://www.ncdc.noaa.gov/data-access/land-based-station-data/land-based-datasets/global-historical-climatology-network-ghcn), coordinated by National Climatic Data Center at NOAA,
+-   The [Daymet](https://daymet.ornl.gov/) gridded estimates of daily weather parameters for North America, version 3, available from the Oak Ridge National Laboratory's Distributed Active Archive Center (DAAC), and
+-   The [International Tree Ring Data Bank (ITRDB)](http://www.ncdc.noaa.gov/data-access/paleoclimatology-data/datasets/tree-ring), coordinated by National Climatic Data Center at NOAA.
+
+FedData is designed with the large-scale geographic information system (GIS) use-case in mind: cases where the use of dynamic web-services is impractical due to the scale (spatial and/or temporal) of analysis. It functions primarily as a means of downloading tiled or otherwise spatially-defined datasets; additionally, it can preprocess those datasets by extracting data within an area of interest (AoI), defined spatially. It relies heavily on the [**sp**](https://cran.r-project.org/package=sp), [**raster**](https://cran.r-project.org/package=raster), and [**rgdal**](https://cran.r-project.org/package=rgdal) packages.
```

Comment:
put all package names in backticks, no bold
e.g. [`raster`](https://cran.r-project.org/package=raster)